lib/modules/versioning/redhat/readme.md

```diff
@@ -0,0 +1,5 @@
+Red Hat versioning is used with container images that are maintained by Red Hat.
+
+Corresponding definitions start with the version of the main component, followed by a hyphen, and a release/build version.
+
+Ranges are not yet supported by this versioning.
```

Comment:
```suggestion
Ranges are not supported by this versioning.
```

Saying `yet` implies that it will get range support later. Do we already know if somebody is going to work on range support?

Reply:
You're right. As of now, no signs of ranges or any plans known that they would work on them.